pyproject.toml

```diff
@@ -44,3 +44,6 @@ profile = "black"
 [build-system]
 requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
+
+[tool.pytest.ini_options]
+addopts = "-v --cov=components --cov-branch --cov-report=term-missing"
```

Comment:
Missing line break.

Also: I think there should be a dev-dependency to `pytest` here.